container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -480,7 +480,7 @@ instance_groups:
     release: loggregator-agent
   configuration:
     templates:
-      properties.dns_health_check_host: 'mysql-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))'
+      properties.dns_health_check_host: 'mysql-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN)).'
```

Comment:
Why has the host to end in a `.` ?

Reply:
See the [commit message](https://github.com/SUSE/scf/pull/1864/commits/18004ca202d870e00230b81d1adb391ea2b85c69) — the scripts run `host -r <dns_health_check_host>`, and that appears to not work because it tries to ask upstream resolvers (instead of asking the kube internal one), which eventually times out.  It's an issue with the `ndots` setting in `/etc/resolv.conf` in the container, I think, but we don't control that.

Reply:
Yeah, adding the final dot makes it unambiguously a fully qualified domain name:
> Since a complete domain name ends with the root label, this leads to a printed form which ends in a dot.

from https://www.ietf.org/rfc/rfc1034.txt

Reply:
ty. resolved for me